rts/Sim/Weapons/DGunWeapon.cpp

```diff
@@ -12,7 +12,8 @@ CR_REG_METADATA(CDGunWeapon, )
 
 void CDGunWeapon::FireImpl(const bool scriptCall)
 {
-	float3 dir = wantedDir;
+	float3 dir = currentTargetPos - weaponMuzzlePos;
+	const float dist = dir.LengthNormalize();
```

Comment:
`dist` looks unused here

Reply:
Good catch. I had just copied over from LaserCannon.cpp, where it does use dist later. 
But DGunWeapon.cpp does not need that value.